_posts/2022-9-5-Diplomat @ Google.md

```diff
@@ -62,10 +80,11 @@ borrowed opaques, string slices, and slices of primitives.
 3. Can only cross the FFI boundary by value, meaning that every field is also
 converted at the boundary.
 
-Additionally, Diplomat also supports fieldless enums, which are just passed
-as their discriminents across the boundary and then converted appropriately.
+In JavaScript, for example, "bag 'o stuff" structs are converted into classes
```

Comment:
this is when returning, when accepting JS will accept any JS object with fields named the right way